app/src/main/res/values-nl/strings.xml

```diff
@@ -0,0 +1,259 @@
+    <string name="uses_aosp_detail">Support-v4 27.1.0, Appcompat-v7 27.1.0, RecyclerView 27.1.0, CardView 27.1.0, Design 27.1.0, Palette 27.1.0, Preference-v14 27.1.0, Room 1.1.0; gedistributeerd door het Android Open Source Project onder de Apache License, versie 2.0</string>
+    <string name="uses_exoplayer_detail">Versie 2.6.1; gedistributeerd door Google Inc. onder een Apache License, versie 2.0</string>
+    <string name="uses_libflac_detail">Version 1.3.1; distributed by the Xiph.Org Foundation under a BSD License</string>
+    <string name="uses_dagger_detail">Versie 2.6; gedistributeerd door Google Inc. en Square Inc. onder een Apache License, versie 2.0</string>
+    <string name="uses_rx_java_detail">Versie 1.6.1; gedistributeerd door Netflix, Inc. onder een Apache License, versie 2.0</string>
+    <string name="uses_rx_android_detail">Versie 1.2.1; gedistributeerd door RxAndroid onder een Apache License, versie 2.0</string>
+    <string name="uses_rx_permissions_detail">Versie 0.7.0; gedistributeerd door Thomas Bruyelle (tbruyelle) onder een Apache License, versie 2.0</string>
+    <string name="uses_retrofit_detail">Versie 2.0.2; gedistributeerd door Square, Inc. onder een Apache License, versie 2.0</string>
+    <string name="uses_gson_detail">Versie 2.7; gedistributeerd door Google, Inc. onder een Apache License, versie 2.0</string>
+    <string name="uses_glide_detail">Versie 3.7.0; gedistributeerd door Google, Inc. onder een BSD licensie</string>
+    <string name="uses_vert_slider_detail">Versie 0.5.2; gedistributeerd door Haruki Hasegawa onder een Apache License, versie 2.0</string>
+    <string name="uses_fastscroll_detail">Versie 1.0.16; gedistributeerd door Tim Malseed onder een Apache License, versie 2.0</string>
+    <string name="uses_seek_arc_detail">Versie 1.1; gedistributeerd door Triggertrap Ltd (Neil Davies) onder een MIT License</string>
+    <string name="uses_timber_detail">Versie 1.1; gedistributeerd door Triggertrap Ltd (Neil Davies) onder een MIT License</string>
+    <string name="about_description">Jockey is een muziekspeler voor Android 4.1 en hoger, gefocused op simpliciteit, snelheid en design gedistributeerd onder een Apache 2.0 licentie.</string>
+    <string name="empty_included_dirs_detail">Jockey kan beperkt worden om in alleen specifieke mappen te doorzoeken, druk op de “+” knop om een map in te sluiten.</string>
+    <string name="empty_excluded_dirs_detail">Jockey kan beperkt worden om sommige mappen over te slaan, druk op de “+” knop om een map uit te sluiten.</string>
+    <string name="url_privacy_policy">https://www.marverenic.com/Jockey/privacy-policy.html</string>
+    <string name="first_launch_detail">"<![CDATA[Door deze applicatie te gebruiken ga je akkoord met Jockey\'s         <a href="https://www.marverenic.com/Jockey/privacy-policy.html">privacybeleid</a>.]]>     "</string>
+    <string name="pref_resume_on_connect">Speel af als een hoofdtelefoon wordt verbonden</string>
```

Comment:
This header is actually getting truncated on my Pixel XL. Is there a way to rephrase this to be more concise? It doesn't have to translate back to "Play when plugging in headphones", just as long as it's able to describe the concept of "Music will be played when you plug in any pair of headphones".

Reply:
This is the shortest grammatically correct sentence I can think of.

I'll change it in the next commit to "Afspelen als headset verbonden", which is not grammatically correct, but the concept of "Music will be played when you plug in any pair of headphones" will still be clear.